app-misc/vcontrold/files/vcontrold

```diff
@@ -0,0 +1,7 @@
+#!/sbin/openrc-run
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+PIDFILE="/run/vcontrold.pid"
```

Comment:
'pidfile' goes lowercase, and EXEC is not used at all now. Use `man runscript`.

Reply:
(Hopefully) Fixed in 26ed3d2dfa633955214a2672405047181f0943c0

Reply:
Apparently, setting `command` and doing nothing else is already enough?!

Reply:
I don't know, you are supposed to test it ;-). If it works good for you (starting and stopping), I'm all for the less.